jb/itch5/order_book_depth.hpp

```diff
@@ -97,23 +101,27 @@ class order_book {
   }
 
  private:
-  /**
-   * Refactor handle_add_order()
+ /**
+   * Refactor handle_add_order().
    *
+   * Increments order_book_ if a new price (px) is emplaced into the map.
+   * Generates a false positive when qty is 0.
    * @tparam book_side the type used to represent the buy or sell side
    * of the book
    * @param side the side of the book to update
    * @param px the price of the new order
    * @param qty the quantity of the new order
-   * @returns true if the inside changed
+   * @return true if the inside changed
```

Comment:
Both \return and \returns (or the versions with @) are valid:

https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdreturns